lessons/Data-Science/ai-linear-regression/langs/en-us/pages/alvinn-training.adoc

```diff
@@ -0,0 +1,61 @@
+= Supervised Machine Learning: Training ALVINN
+
+@n Imagine that ALVINN has trained on the 15 images collected during a half minute of driving. How do you predict its best guesses about the steering angle will be? Explain. @fitb{}{}
```

Comment:
Personally, I don't feel like I know what the answer would be, other than "not very good". The fact that there's this paragraph worth of space is kind of intimidating.

Maybe a short fitb that has kids give their answer about how accurate it will be, then use the rest of the blanks to ask them to justify _why_ they think that.